README.md

```diff
@@ -54,18 +54,13 @@ export GOOGLE_APPLICATION_CREDENTIALS=<absolute path to sa-key.json>
 
 ## Manual Testing
 
-- Fork this repo, then push your feature branch there.
-- On your fork, cut a release from that feature branch, e.g. USER-v1
+- Push your topic branch to GitHub.
 - From your notebook, run:
 
   ```
-  !pip install --user --upgrade 'https://github.com/USER/pyclient/archive/pyclient-USER-v1.zip#egg=aou_workbench_client&subdirectory=py'
+  !pip install --user --upgrade 'git+https://github.com/all-of-us/pyclient.git@my-topic-branch#egg=aou_workbench_client&subdirectory=py'
```

Comment:
You should mention to use the appropriate version of pip here based on 2.7 vs. 3.0.

Also, they may need to restart the kernel once this is done... I'm not certain.

Reply:
Good point. Done.

I tested out the kernel restart. Minimally it seems necessary if you've already imported the client library. Added a note here.